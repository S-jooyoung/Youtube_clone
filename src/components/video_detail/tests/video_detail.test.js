//import dependencies
import React from "react";

//import custom-component
import VideoDetail from "../video_detail";

// import snapshot-testing methods
import renderer from "react-test-renderer";

describe("VideoDetail", () => {
  const video = {
    id: "cheYIVEtVQ4",
    snippet: {
      title: "Test Title",
      description: "Test Desciption",
      channelTitle: "TestCode is good",
      defaultAudioLanguage: "en",
    },
  };

  it("renders", () => {
    const component = renderer.create(<VideoDetail video={video} />).toJSON();

    expect(component).toMatchSnapshot();
  });
});
