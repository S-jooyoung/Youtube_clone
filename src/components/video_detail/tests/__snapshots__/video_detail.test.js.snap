// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoDetail renders 1`] = `
<section
  className="detail"
>
  <iframe
    allowFullScreen={true}
    className="video"
    frameBorder="0"
    height="500"
    src="https://www.youtube.com/embed/cheYIVEtVQ4"
    type="text/html"
    width="100%"
  />
  <h2>
    Test Title
  </h2>
  <h3>
    TestCode is good
  </h3>
  <pre
    className="description"
  >
    Test Desciption
  </pre>
</section>
`;
